lib/shared/link_information.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:thunder/core/enums/media_type.dart';
+import 'package:thunder/core/enums/view_mode.dart';
+
+class LinkInformation extends StatefulWidget {
+  final ViewMode viewMode;
```

Comment:
Just for sake of clarity, I would add some comments to all of the parameters that are being passed in!

Reply:
Added in latest commit!